bcos-executor/src/executive/BlockContext.cpp

```diff
@@ -45,16 +45,16 @@ using namespace std;
 BlockContext::BlockContext(std::shared_ptr<storage::StateStorageInterface> storage,
     LedgerCache::Ptr ledgerCache, crypto::Hash::Ptr _hashImpl,
     bcos::protocol::BlockNumber blockNumber, h256 blockHash, uint64_t timestamp,
-    uint32_t blockVersion, const VMSchedule& _schedule, bool _isWasm, bool _isAuthCheck,
+    uint32_t blockVersion, bool _isWasm, bool _isAuthCheck,
     storage::StorageInterface::Ptr backendStorage)
   : m_blockNumber(blockNumber),
     m_blockHash(blockHash),
     m_timeStamp(timestamp),
     m_blockVersion(blockVersion),
-    m_schedule(_schedule),
     m_isWasm(_isWasm),
     m_isAuthCheck(_isAuthCheck),
     m_storage(std::move(storage)),
+    m_transientStorageMap(std::make_shared<transientStorageMap>(10)),
```

Comment:
why not use std::map?
the transientStorage's lifetime is tx, every tx has its transientStorage. I think we don't need to consider parallel